lib/fs_utils/asset.js

```diff
@@ -58,14 +57,12 @@ class Asset {
       debug(`Copied ${path}`);
       this.updateTime();
       this.error = null;
-      return Promise.resolve();
     }, err => {
       debug(`Cannot copy ${path}: ${err}`);
       this.updateTime();
       const error = new Error(err);
       error.code = 'Copying';
-      this.error = error;
-      return Promise.reject(this.error);
+      throw this.error = error;
```

Comment:
Can we separate this in two lines? May be very unusual for outside contributors.